_test/test_change_crystal/test_change_crystal_file_eq_memory.m

```diff
@@ -0,0 +1,138 @@
+classdef test_change_crystal_file_eq_memory < TestCase
+    % Perform tests of change_crystal functions and methods.
+    %
+    %
+    % It is assumed that test_change_crystal_1 has already been successfully performed.
+    % That test ensures that change_crystal_sqw on a file is correct; here we check that
+    % all the permutations of object types and files are handled correctly.
+    %
+    % Author: T.G.Perring
```

Comment:
as this is a brand new file, querying if toby is the author. If we need to have a previous test succeed, is it possible to chain them so fail-1 implies fail-2?

Reply:
This is refactoring of `test_change_crystal_2` written by Toby. I just gave the file meaningful name and have thrown away all Toby's palaver, hiding the purpose of the test.

Not sure what you mean --despite we know the order tests are running, according unit tests good practice, each unit test should be independent on others. Toby probably meant that the correctness of change itself is verified elsewhere, and here we just verify equivalence between  file ane memory change.